app/RouteHandlers/AddDonationHandler.php

```diff
@@ -221,4 +222,15 @@ private function newTrackingInfoFromRequest( Request $request ): DonationTrackin
 	private function filterAutofillCommas( string $value ): string {
 		return trim( preg_replace( ['/,/', '/\s{2,}/'], [' ', ' '], $value ) );
 	}
+
+	/**
+	 * Since the application has no concept of logging out or resetting a session, we reset any session cookies at this point
+	 *
+	 * @param Response $response
+	 * @return Response
+	 */
+	private function resetSessionCookies( Response $response ) {
```

Comment:
I found this method a bit confusing, because it's not clear *why* we're resetting the cookie (because we want to differentiate between two "paths" to the confirmation page)